kernel/dev/vga/graphics.cpp

```diff
@@ -97,9 +131,13 @@ void init(FramebufferInfo info)
     {
         map_kernel_page(VADDR(page), page);
     }
+    // Alloc the backbuffer
+    backbuffer = ((void*) (malloc(height* pitch)));
```

Comment:
`backbuffer = (void*)malloc(height * pitch);`

Reply:
That cast is unneeded, the signature is already `void *`